package.json

```diff
@@ -105,6 +105,12 @@
     "webpack-cli": "4.9.2",
     "webpack-dev-server": "4.9.0"
   },
+  "overrides": {
+    "faye-websocket": "^0.11.4",
+    "@types/mime": "^3.0.1",
+    "is-unicode-supported": "^1.3.0",
+    "lodash": "$lodash"
```

Comment:
This is here because `lodash` is already versioned above and npm does not like 2 references with the same number